# DepthImageInfo.msg
sensor_msgs/Image rgb_image
sensor_msgs/Image depth_image
voxseg/TransformationMatrix cam_extrinsics
